lua/cmp/core.lua

```diff
@@ -72,18 +72,30 @@ end
 ---@type table<number, cmp.Source>
 core.sources = {}
 
+---@type table<string, table<cmp.Source>>
```

Comment:
I prefer to annotate by `cmp.Source[]` instead.

Reply:
Not sure what you mean.

Reply:
ok got it...

Reply:
You should use `---@type table<string cmp.Source[]>` instead.